app/frontend/src/components/objectList/ObjectPermissions.vue

```diff
@@ -9,7 +9,7 @@
     >
       <template #title>
         <v-icon color="primary" class="mr-5"> mdi-account-group </v-icon>
-        Permissions for <span class="ml-1 font-weight-bold">{{ objectName }}</span>
+        Permissions for file: <span class="ml-1 font-weight-bold">{{ objectName }}</span>
```

Comment:
I think we should avoid calling things files, even though they basically are, across all of COMS/COMSS if possible. Our product focuses on objects and object manipulation, so we should continue to emphasize that. I recommend reverting this, or changing it to `object:` instead if the noun must be described.